.github/workflows/pr-builder.yml

```diff
@@ -169,7 +174,23 @@ jobs:
 
       - name: 🃏 Run Jest & Collect Coverage
         id: run-jest-test-and-coverage
-        run: pnpm test
+        run: pnpm test:ci
+
+      - name: Upload `@asgardeo/js` coverage reports to Codecov
+        id: upload-asgardeo-js-coverage
+        uses: codecov/codecov-action@v4.0.1
+        with:
+          files: ./packages/core/coverage/coverage-final.json
+          flags: '@asgardeo/js'
```

Comment:
@brionmario if are using flags , lets add the necessary flag paths in the codecov yaml as well.
Referhttps://docs.codecov.com/docs/flags

Reply:
Updated.